### PR TITLE
Add button to bulk import assessments from file system

### LINKED ISSUE #1928: Bulk import assessments from file system
Request from 122:
> Under "Install Assessment > Import from file system", provide an "Import all" option. Having to click on individual assignments made sense when one had to configure the dates (pretty much everything else was in .yml file) but now there is no reason to import them one by one.

As a workaround, the following javascript code can be entered into the browser console to "click" all assessments:
`$(".section a[rel='nofollow']").each((i, e) => $.post(e.href))`

### PR DESCRIPTION
**Pull request by damianhxy**
### Summary generated by Reviewpad on 16 Jul 23 19:01 UTC
This pull request includes several patches that make various changes to the codebase. Here is a summary of each patch:

1. [PATCH 1/7] Update old-style path to use path helper:
   - Makes a modification in the assessments_controller.rb file to update the old-style path to use the path helper. This change updates the redirect_to method call.

2. [PATCH 2/7] Add install all button:
   - Makes changes in the install_assessment.html.erb file to add an "Install all" button. The button triggers a script that sends a POST request for each import link and reloads the page after the import is done.

3. [PATCH 3/7] Rename install -> import:
   - Renames the "install" class to "import" in the install_assessment.html.erb file.

4. [PATCH 4/7] Improve logic:
   - Adds logic to display an alert message when the "Import all" button is clicked. The message indicates whether all assessments have been imported successfully, or if there were errors during the import process.

5. [PATCH 5/7] Add alert:
   - Adds an alert message to inform the user about the completion of the import process and any errors encountered during the import of assessments.

6. [PATCH 6/7] Rephrase message:
   - Modifies the alert message to rephrase the text when all assessments have been imported, but there are remaining assessments that encountered errors during the import process.

7. [PATCH 7/7] Rename btn variable:
   - Renames the 'btn' variable to '$btn' for clarity in the install_assessment.html.erb file.

Please review these patches and provide any necessary feedback or comments.

## Description

- Update an old-style path to use a path helper
- Improve UI of uninstalled assessments list
- Add button to bulk import assessments

## Motivation and Context

Currently, assessments can only be individually imported from the file system. This PR provides an option to import all of them at once.

Closes #1928

## How Has This Been Tested?

Create a course and a few assessments. Delete all of the assessments.
- Ensure that importing from file system still works
- Ensure that import all button works

Screenshot 2023-06-25 at 02 58 16

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have run rubocop for style check. If you haven't, run `overcommit --install && overcommit --sign` to use pre-commit hook for linting
- [ ] My change requires a change to the documentation, which is located at [Autolab Docs](https://github.com/autolab/docs)
- [ ] I have updated the documentation accordingly, included in this PR

### CODE REVIEW

**Review by evanyeyeye — CHANGES_REQUESTED** [review]
Functionally everything works. However, on the frontend side the **Import All** button seems to remove only one assessment from the list per click, when it should remove them all. 

To reproduce this:
- Add and remove 2 assessments.
- Click **Import All**. 
- Observe that only one assessment is removed from the list, but that both are added to the course. If you refresh the page or click the button again, the list will be empty.

**Comment by damianhxy** [comment]
Most likely the reload is happening so quickly that the assessments didn't finish importing in time.

I'll have to modify the code so that all ajax requests are completed before reloading the page.

ref: https://stackoverflow.com/questions/3709597/wait-until-all-jquery-ajax-requests-are-done

**Comment by damianhxy** [comment]
I've improved the javascript code so that the page only reloads after all ajax requests are complete

File: app/views/assessments/install_assessment.html.erb [hunk 1]
```diff
@@ -1,5 +1,29 @@
...
+        const btn = $(e.target);
+        btn.text("Importing...");
+        btn.addClass("disabled");
+
+        Promise.all(
+            $(".import").map((i, e) => $.post(e.href))
+          )
+          .then(() => {
+            btn.text("Done!");
+            location.reload();
```
Comment (20wildmanj) [inline app/views/assessments/install_assessment.html.erb:20]: Any easy way to add some sort of visual indicator that the imports finished successfully? Gonna assume it's hard to use a flash, but the "Done!" shows up for ~100ms so it's pretty hard to notice. If it's not easy to do, nbd

**Review by 20wildmanj — APPROVED** [review]
LGTM besides comment on post-bulk-install notification

File: app/views/assessments/install_assessment.html.erb (hunk 1 above)
Comment (damianhxy) [reply app/views/assessments/install_assessment.html.erb:20]: Added alert when imports finish

**Pull request merged**